### PR TITLE
Create config migration

### PR DESCRIPTION
**Pull request by diegoalzate**
references https://github.com/Rpc-h/degen-rpch-net/issues/52

### overview
```
creates configs table 

key (PK)
data (not null)
created_at
updated_at
```

### CODE REVIEW

File: apps/discovery-platform/migrations/1697029706619_add-configs-table.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable camelcase */
+
+exports.up = (pgm) => {
+  pgm.createTable("configs", {
+    key: { type: "text", primaryKey: true },
```
Comment (esterlus) [inline apps/discovery-platform/migrations/1697029706619_add-configs-table.js:5]: lets make the primary key fixed width with `varchar(255)` or something similar

File: apps/discovery-platform/migrations/1697029706619_add-configs-table.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    data: { type: "text", notNull: true },
+    created_at: "createdAt",
+    updated_at: "timestamp",
+  });
+};
+
+exports.down = (pgm) => {
```
Comment (esterlus) [inline apps/discovery-platform/migrations/1697029706619_add-configs-table.js:12]: createTable can automatically infer down migration. you can remove this section entirely.

**Review by esterlus — CHANGES_REQUESTED** [review]
only minor improvements requested

File: apps/discovery-platform/migrations/1697029706619_add-configs-table.js (hunk 2 above)
Comment (diegoalzate) [reply apps/discovery-platform/migrations/1697029706619_add-configs-table.js]: cool

**Review by esterlus — APPROVED** [review]

**Pull request merged**